examples/get_headers.rb

```diff
@@ -0,0 +1,10 @@
+Shoes.app do
```

Comment:
this is to get headers and other stuff

Reply:
![Screenshot_20230710_134132](https://github.com/scarpe-team/scarpe/assets/85360080/2324d2f9-aa8b-4fe6-a723-994c651073be)